### PR TITLE
📝 add progress documentation

### PR DESCRIPTION
**Pull request by YounesBessa**

### CODE REVIEW

**Review by jerembdn — APPROVED** [review]

File: docs/pages/docs/components/progress.mdx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+<Playground
+  code={`
+() => {
+const [value1, setValue1] = React.useState(0);
+const { theme } = useTheme();
+return (
+<Container>
+  <Progress
+    value={value1}
```
Comment (AntoineKM) [inline docs/pages/docs/components/progress.mdx:46]: value1 => value

File: docs/pages/docs/components/progress.mdx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  <Progress value={30} max={40} />
+`}
+/>
+
+## Dynamic colors
+
+<Playground
+  code={`
+() => {
+const [value1, setValue1] = React.useState(0);
```
Comment (AntoineKM) [inline docs/pages/docs/components/progress.mdx:41]: value1 => value
setValue1 => setValue

File: docs/pages/docs/components/progress.mdx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    }}
+    states={{
+      0: "Your package is in our hands",
+      25: "It is in our network",
+      50: "Your package has arrived at its delivery site",
+      75: "We are preparing your package for delivery",
+      100: "Your package has been delivered",
+    }}
+  />
+  <Container gap={5} row>
```
Comment (AntoineKM) [inline docs/pages/docs/components/progress.mdx:97]: gap={"tiny"}

File: docs/pages/docs/components/progress.mdx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+## States
+
+<Playground
+  code={`
+() => {
+const [value2, setValue2] = React.useState(0);
+const { theme } = useTheme();
+return (
+<Container>
```
Comment (AntoineKM) [inline docs/pages/docs/components/progress.mdx:79]: gap={"small"}

**Review by AntoineKM — APPROVED** [review]
can you fix the background of the progress bar ?

**Comment by YounesBessa** [comment]
> can you fix the background of the progress bar ?

I tried but I can't seem to figure out why it's not changing in the "dynamic colors" example

**Comment by AntoineKM** [comment]
You need to create another styled component `styled.progress` with a prop `background` to be effective

**Review by AntoineKM — APPROVED** [review]

**Pull request merged**